Workbooks/HDInsight/Overview/Overview.workbook

```diff
@@ -0,0 +1,442 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "# HDInsight Kafka Insights"
+      },
+      "name": "text - 1"
+    },
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "query": "",
+        "crossComponentResources": [
+          "{Workspace}"
+        ],
+        "parameters": [
+          {
+            "id": "741603f6-06d5-431c-b989-7e53c252cda7",
+            "version": "KqlParameterItem/1.0",
+            "name": "Subscription",
+            "type": 6,
+            "isRequired": true,
+            "value": "/subscriptions/d66b1168-d835-4066-8c45-7d2ed713c082",
```

Comment:
items with actual subscription/resource ids won't work for people that don't have access to these specific resources

Reply:
fixed, thanks. I think this was just the currently selected one when I copied the workbook JSON